src/pages/Home/components/Banner/styles.module.scss

```
@@ -0,0 +1,22 @@
+.homeMainImage {
+  width: 100vw;
+  height: 25rem;
+  background: linear-gradient(0deg, rgba(2,0,36,1) 0%, rgba(3,3,3,0.6927309782608695) 35%, rgba(255,255,255,0) 100%),url('./src/assets/home-food-bg.jpg') ;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #d3d3d3;
+
+  &Title {
```

Именуем классы по БЭМу, parent__child_mod
```suggestion
  &__Title {
```

---

done